assets/src/edit-story/components/library/panes/shapes/shapesPane.js

```diff
@@ -71,29 +75,34 @@ function ShapesPane(props) {
       <Section title={__('Basic shapes', 'web-stories')}>
         <SectionContent>
           {/** Basic masks */}
-          {MASKS.map((mask) => (
-            <ShapePreview
-              key={mask.type}
-              onClick={() => {
-                insertElement('shape', {
-                  backgroundColor: createSolid(51, 51, 51),
-                  width: DEFAULT_ELEMENT_WIDTH,
-                  mask: {
-                    type: mask.type,
-                  },
-                });
-              }}
-              alt={mask.name}
-            >
-              <svg
-                viewBox={'0 0 1 1'}
-                width={PREVIEW_SIZE}
-                height={PREVIEW_SIZE}
+          {MASKS.map((mask) => {
+            const widthMultiplier = mask.ratio > 1 ? 1 / mask.ratio : 1;
```

Comment:
Since we control the shapes, can't we just fix the ratio to be =width/height? That way we don't have to check if it's smaller/bigger than 1

Reply:
We should just always use the same `width == DEFAULT_ELEMENT_WIDTH` and only ratio the `height`. The `insertElement` will do the rest for fitting onto canvas.

Reply:
SGTM. Note that this will mean that the preview is not fitted into a box (contain logic), but instead, the width of the preview is always 36, and the height can be larger or smaller. But I think I can work around it in CSS.

Reply:
To clarify, I don't know why I said what I said. It's totally ok to based the calculations on height and let width be ratio-ed. My main point still stands: we don't need to worry about calculations producing too big of a width or height for a canvas - `insertElement` takes care of this.